paperqa/llms.py

```diff
@@ -129,9 +129,9 @@ class LLMModel(ABC, BaseModel):
         " LLMResult (different than callbacks that execute on each chunk)",
         exclude=True,
     )
-    config: dict = Field(default={})
+    config: dict = Field(default_factory=dict)
 
-    async def acomplete(self, prompt: str) -> str:
+    async def acomplete(self, prompt: str) -> tuple[str, tuple[int, int]]:
```

Comment:
Since this is sort of an `abstractmethod`, can you somehow document what the return type is?

Can make a `NamedTuple` that is sort of self-documenting, or a docstring

Reply:
OK added a new class and revised code to use it

Reply:
Nice, thanks